pkg/daemon/server/daemon_server.go

```diff
@@ -47,6 +51,14 @@ import (
 	"github.com/numaproj/numaflow/pkg/watermark/fetch"
 )
 
+var (
```

Comment:
It would be better to move all the metrics related items to a new file `metrics.go`.